Makefile

```diff
@@ -32,7 +32,7 @@ test:
 	go tool cover -html="coverage.txt" -o "coverage.html"
 
 integration_test:
-	go test -count=1 -race -covermode=atomic -v ./tests
+	STORAGE_OSS_INTEGRATION_TEST=on go test -count=1 -race -covermode=atomic -v ./tests
```

Comment:
We don't need to set `STORAGE_OSS_INTEGRATION_TEST=on` here, we will use `Makefile.env`